scss/components/_fonts.scss

```diff
@@ -0,0 +1,75 @@
+@font-face {
```

Comment:
Actually this file is more related to icons than to fonts, so suggest to rename it into `icons`.